File: dataset/human/python/file_304.py
from fractions import gcd
def get_result(L, N):
    L_abs = [abs(x) for x in L]
    A = sum(L_abs)
    B = min(L_abs)
    neg_count = 0
    for elem in L:
        if elem < 0:
            neg_count += 1
    if neg_count % 2 == 0:
        return A
    else:
        return (A-2*B)


N = int(input())
L = [int(s) for s in input().split(" ")]
result = get_result(L, N)
print(result )

File: dataset/human/c/file_304.c
#include <stdio.h>
#include <stdlib.h>

int main ( )
{
  int N, i, min, cnt;
  long sum;
  long A[ 100000 ];
  scanf ( "%d", &N );
  sum = 0;
  min = 1000000000;
  cnt = 0;
  for ( i = 0 ; i < N ; i++ )
    {
      scanf ( "%ld", &A[ i ] );
      if ( A[ i ] < 0 )
	{
	  cnt++;
	  A[ i ] = abs ( A[ i ] );
	}
      if ( min > A[ i ] )
	min = A[ i ];
      sum += A[ i ];
    }
  if ( cnt % 2 )
    sum -= min * 2;
  printf ( "%ld\n", sum );
  return 0;
}
